.gitignore

```diff
@@ -156,4 +156,10 @@ dmypy.json
 cython_debug/
 
 # VSCode
-.vscode
\ No newline at end of file
+.vscode
+
+
+# Demo data
+docs/notebooks/bag_light_AMS_WGS84.gpkg
+docs/notebooks/stm.zarr
+docs/notebooks/stm_ordered.zarr
```

Comment:
```suggestion
```
I would suggest let's not add these files to `.gitignore`. We do not have plans to host these example datasets in the repo.
IMO, for development purposes, I think people should be able to see the file they generated in git rather than ignore them.